core/tabs/tabs.py

```diff
@@ -48,3 +50,25 @@ def tab_move_right():
     # def tab_duplicate():
 
     # def tab_back():
+
+    # requires: https://github.com/gcmt/taboo.vim/
+    def tab_rename(text: str = ""):
+        actions.user.vim_run_command_exterm(f":TabooRename {text}")
+
+    def tab_new_named(text: str = ""):
```

Comment:
tab_open_with_name

Reply:
I'll start porting over the tab/split names from the community PR into neovim-talon again, and then once that is merged, these can be updated